interview-questions.md

```diff
@@ -8,32 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Compare and contrast arrays and hashes in Ruby. Include at least two differences and two similarities.
 
-Your answer:
+Your answer: They're both collections of data. 
```

Comment:
This answer should demonstrate how you would respond to the question in an interview.  Providing very little content or response to the interviewer will not help you stand out in a good way.  Your interviewer is looking at both your technical knowledge and also how you communicate about it.  These questions should always try to display the knowledge you know about the content.  If you are struggling to come up with context, provide an example of each that demonstrates your understanding of their usage.